src/App.tsx

```diff
@@ -10,9 +11,17 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<h1 className="title">Home Page</h1>} />
+            <Route path="/home" element={<Navigate to="/" replace={true} />} />
+            <Route path="/people">
+              <Route path=":personSlug?" element={<PeoplePage />} />
+            </Route>
```

Comment:
The nested route for `/people` should be wrapped correctly to handle direct access without a slug. Consider using a `Route` element with a path of `/people/*` to ensure proper routing.